### ISSUE TITLE
FIX: ESLINT: No-Static-Element-Interactions

### ISSUE THREAD

**Issue by karlchvojka** [karlchvojka/reactjs_boilerplate#1]
# Context
ESLinter is a linter configuration for the ReactJS boilerplate that is present in each flavour. 

## Description
Error in vscode: templates\reactjs_template_2022.eslintrc.js: Configuration for rule "jsx-a11y/no-static-element-interactions" is invalid: Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"always"').

## Solution
./.eslintrc.js:
- Line 27: Change sevarity rule -> 2 (error)

**Comment by karlchvojka** [comment]
Completed with branch reactjs_base/1-no-static-element-interactions
